utils/graphql/SDL.graphql

```diff
@@ -0,0 +1,151 @@
+type Query {
+  """A Single Transaction by its ID"""
+  transactionById(id: Int): Transaction
+
+  """List of All Transactions"""
+  transactions: [Transaction]
```

Comment:
everywhere in the graphql schema, transaction refers to TNUA in Hashavshevet. a couple of transactions are a title in the schema - KOTERET in Hashavshevet. MANA is batch in the schema.
In most of bank and credit card data Koteret is called transaction so Hashavshevet really mess things up (and even in their own documentation they mix Tnua and Koteret).
We should add semantics attribute in the documentation, and maybe change it a little so it would be very clear.